src/index.html

```diff
@@ -22,6 +22,421 @@
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+          </div>
+          <div class="card__rate-review">
+            <p class="card__rate-review-p">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-numb">$2,199</p>
+        </div>
+
+        <div class="card__block">
+          <button
+            class="card__block-button"
+            data-qa="card-hover"
+          >
+            Buy
+          </button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+        <div class="card__title">
+          <p class="card__title-p">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+        <div class="card__code">
+          <p class="card__code-p">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="card-stars card-stars--4">
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+          </div>
+          <div class="card__rate-review">
+            <p class="card__rate-review-p">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-numb">$2,199</p>
+        </div>
+
+        <div class="card__block">
+          <button
+            class="card__block-button"
+            data-qa="hover"
+          >
+            Buy
+          </button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+        <div class="card__title">
+          <p class="card__title-p">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+        <div class="card__code">
+          <p class="card__code-p">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="card-stars card-stars--4">
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+          </div>
+          <div class="card__rate-review">
+            <p class="card__rate-review-p">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-numb">$2,199</p>
+        </div>
+
+        <div class="card__block">
+          <button
+            class="card__block-button"
+            data-qa="hover"
```

Comment:
The 'data-qa' attribute here is 'hover', which is inconsistent with the first card button where it is 'card-hover'. Ensure consistency in attribute naming to avoid confusion.